### ISSUE TITLE
CCEffect node acts weirdly when added under BoxLayout, node it content size changes

### ISSUE THREAD

**Issue by khanbhai89** [spritebuilder/SpriteBuilder#1142]
1. Launch Application.
2. Delete everything from MainScene.ccb.
3. Add a CCBoxLayout on to the stage.
4. Add CCEffect under it.
5. Add two Color Nodes under CCEffect Nodes.
6. Change Content Size of CCEffect dimensions from Points to "%".
   
   -- Result: Value of the content size is changed to "1" for both height and width. Even when we increase the size to 100% for both it doesn't display.
   
   This happened 3/3 times.

![screen shot 2014-11-18 at 10 09 27 pm](https://cloud.githubusercontent.com/assets/7450007/5092282/b3aaaad2-6f6f-11e4-91e5-f74a45b99e49.png)

Mac OS X: 10.9.5.
XCode: 6.0.

**Comment by nsluke** [comment]
I think this may not be a bug with sprite builder. The problem seen here is that if you create a layout box and then do not provide any context for the size of the objects inside of it, you will get some weird results. 

A layout box has inferred dimensions, which is why you can't change the size of it. It's size is completely dependent on the objects you put inside of it. When you put an object inside of a layout box and then say "% scale of parent container" you end up with a interesting paradox. The layout box is trying to get its size from the effect node, and the effect node is trying to get its size from the layout box. Unfortunately, there isn't a fix for this. You need to provide actual ui point dimensions for the objects inside a layout box.

tl;dr: Not a bug. You need to provide actual ui point dimensions for the objects inside a layout box.

**Comment by NoorFatima** [comment]
According to the above comment, this issue is fixed.
Device: Mac OS X 10.9.5
SB Version: 1.4.7
